custom_components/ferroamp/sensor.py

```diff
@@ -531,7 +559,7 @@ async def options_update_listener(hass, entry):
 class FerroampSensor(SensorEntity, RestoreEntity):
     """Representation of a Ferroamp Sensor."""
 
-    def __init__(self, name, unit, icon, device_id, device_name, interval, config_id, **kwargs):
+    def __init__(self, name, slug, unit, icon, device_id, device_name, interval, config_id, **kwargs):
```

Comment:
Maybe change `slug` to `entity_prefix` or send integration name instead and slugify everything at once when setting `entity_id`? I got a bit confused when reading the code as it is now 🙂

Reply:
Good point! I've looked at it myself as well. Changing.